src/conditional-flow/string-conditions.js

```diff
@@ -1,41 +1,78 @@
 // This function should return true if the passed string is equal to "Hello"
-function isHello (val1) {
-
+function isHello(val1) {
   // TODO: write code in this function body to pass the tests
-
+  if (val1 == 'Hello') {
```

Comment:
Same as above, you don't need the if statement here, you can just return `val1 == 'Hello'`